src/coreclr/jit/emitarm64.cpp

```diff
@@ -1500,6 +1500,74 @@ void emitter::emitInsSanityCheck(instrDesc* id)
             }
             break;
 
+        case IF_SVE_JD_4A: // .........xxmmmmm ...gggnnnnnttttt -- SVE contiguous store (scalar plus scalar)
+            elemsize = id->idOpSize();
+            assert(insOptsScalableStandard(id->idInsOpt()));
+            assert(isVectorRegister(id->idReg1()));    // ttttt
+            assert(isPredicateRegister(id->idReg2())); // ggg
+            assert(isGeneralRegister(id->idReg3()));   // nnnnn
+            assert(isGeneralRegister(id->idReg4()));   // mmmmm
+            assert(isScalableVectorSize(elemsize));    // xx
+#ifdef DEBUG
+            if ((id->idIns() == INS_sve_st1h) && (id->idInsOpt() == INS_OPTS_SCALABLE_B))
+            {
+                assert(!"sve_st1h with scalable B is reserved");
+            }
+#endif // DEBUG
+            break;
+
+        case IF_SVE_JD_4B: // ..........xmmmmm ...gggnnnnnttttt -- SVE contiguous store (scalar plus scalar)
+            elemsize = id->idOpSize();
+            assert(insOptsScalableWords(id->idInsOpt()));
+            assert(isVectorRegister(id->idReg1()));    // ttttt
+            assert(isPredicateRegister(id->idReg2())); // ggg
+            assert(isGeneralRegister(id->idReg3()));   // nnnnn
+            assert(isGeneralRegister(id->idReg4()));   // mmmmm
+            assert(isScalableVectorSize(elemsize));    // x
+            break;
+
+        case IF_SVE_JJ_4A:   // ...........mmmmm .h.gggnnnnnttttt -- SVE 64-bit scatter store (scalar plus 64-bit scaled
+                             // offsets)
+        case IF_SVE_JJ_4A_B: // ...........mmmmm .h.gggnnnnnttttt -- SVE 64-bit scatter store (scalar plus 64-bit scaled
+                             // offsets)
+        case IF_SVE_JJ_4A_C: // ...........mmmmm .h.gggnnnnnttttt -- SVE 64-bit scatter store (scalar plus 64-bit scaled
+                             // offsets)
+        case IF_SVE_JJ_4A_D: // ...........mmmmm .h.gggnnnnnttttt -- SVE 64-bit scatter store (scalar plus 64-bit scaled
+                             // offsets)
+        case IF_SVE_JK_4A: // ...........mmmmm .h.gggnnnnnttttt -- SVE 64-bit scatter store (scalar plus 64-bit unscaled
+                           // offsets)
+        case IF_SVE_JK_4A_B: // ...........mmmmm .h.gggnnnnnttttt -- SVE 64-bit scatter store (scalar plus 64-bit
+                             // unscaled offsets)
+            elemsize = id->idOpSize();
+            assert(insOptsScalableStandard(id->idInsOpt()) || insOptsScalable32bitExtends(id->idInsOpt()));
```

Comment:
for what formats we need `insOptsScalableStandard(id->idInsOpt())`?

Reply:
Not all of them. Do you want to be specific with it? We have more specific checks when we call the `emit` functions. For sanity checks, I assumed we didn't have to be as specific.

Reply:
I was not sure if we need `insOptsScalableStandard()` for any of the cases listed above?

Reply:
Ok, actually turns out none them are scalable standard :)